src/models/Settings.js

```diff
@@ -163,7 +163,10 @@ export default Model.extend({
       fn: function() {
         // showPasswordToggle is for OIE only.
         // Used to default showPasswordToggleOnSignInPage to true.
-        return this.options?.features?.showPasswordToggleOnSignInPage !== false;
+        const defaultValue = true;
+        const customizedValue = this.options?.features?.showPasswordToggleOnSignInPage ??
```

Comment:
so many questions marks..

Reply:
I would rather create it own function

Reply:
@pradeepdewda-okta what kind of function?

Reply:
just a wrapper `isCustomized()` ?

Reply:
This is an edge case. The reason I have to do this logic for `customizedValue` is because this setting has a different default value in OIE, so I don't want to use the value that would have been passed to this derived function, `this.settings.get('features.showPasswordToggleOnSignInPage')`. The wrapper would go unused.